pallets/automation-time/src/lib.rs

```diff
@@ -264,6 +266,8 @@ pub mod pallet {
 		BadVersion,
 		UnsupportedFeePayment,
 		CannotReanchor,
+		// Delegation Do Not Exist
+		DelegationDNE,
```

Comment:
Never use three-letter acronym like `DNE`. Nobody knows what that means.

Please rename it to,
`DelegationNonFound`